roles/admin_dashboard_ingress/defaults/main.yml

```diff
@@ -0,0 +1,255 @@
+---
+
+# The ingress class to use for admin dashboard ingresses
+admin_dashboard_ingress_class_name: nginx
+
+# Labels for admin dashboard ingresses
+#   The default labels
+admin_dashboard_ingress_labels_defaults:
+  app.kubernetes.io/managed-by: azimuth-ops
+#   Additional labels
+admin_dashboard_ingress_labels_overrides: {}
+#   The dashboard labels
+admin_dashboard_ingress_labels: >-
+  {{-
+    admin_dashboard_ingress_labels_defaults |
+      combine(admin_dashboard_ingress_labels_overrides)
+  }}
+
+# TLS configuration for admin dashboard ingresses
+#   Indicates whether TLS should be enabled
+admin_dashboard_ingress_tls_enabled: "{{ ingress_tls_enabled | default(True) }}"
+#   TLS-specific annotations
+admin_dashboard_ingress_tls_annotations: "{{ ingress_tls_annotations | default({}) }}"
+#   Wildcard TLS certificate, if available
+admin_dashboard_ingress_tls_wildcard_certificate: >-
+  {{ ingress_tls_wildcard_certificate | default(None) }}
+admin_dashboard_ingress_tls_wildcard_key: >-
+  {{-
+    ingress_tls_wildcard_key
+    if ingress_tls_wildcard_key is defined
+    else undef(hint = 'admin_dashboard_ingress_tls_wildcard_key is required')
+  }}
+#   The TLS secret name when a wildcard certificate is used
+admin_dashboard_ingress_tls_wildcard_secret_name: admin-dashboard-tls
+
+
+# The basic auth username to use for admin dashboards
+admin_dashboard_ingress_basic_auth_username: admin
+# The basic auth password to use for admin dashboards
+admin_dashboard_ingress_basic_auth_password: >-
```

Comment:
People are going to need to add this on upgrade, maybe we need to call this a breaking change?

Reply:
Yes. The release notes stuff I rolled out everywhere specifically respects a `breaking` label that can be added to PRs to make them appear right at the top in their own section